glad/parse.py

```diff
@@ -1,35 +1,27 @@
 from glad.sink import LoggingSink
 
-try:
-    from lxml import etree
-    from lxml.etree import ETCompatXMLParser as parser
-
-    def xml_fromstring(argument):
-        return etree.fromstring(argument, parser=parser())
-    def xml_parse(path):
-        return etree.parse(path, parser=parser()).getroot()
-except ImportError:
-    try:
-        import xml.etree.cElementTree as etree
-    except ImportError:
-        import xml.etree.ElementTree as etree
-
-    def xml_fromstring(argument):
-        return etree.fromstring(argument)
-    def xml_parse(path):
-        return etree.parse(path).getroot()
```

Comment:
This was here for compatibility when there is no lxml installed.

Reply:
Done. I reverted this as it was. The reason I initially dropped xml support was because it's parser lacked [features](https://lxml.de/api/lxml.etree.XMLParser-class.html) such as recovery (now managed by injecting namespaces where needed), entities resolution (which I now handle manually for better control), and the ability to retrieve a node's parent (which I also now do manually).

Now both the lxml and xml modules are supported and produce identical results.